packages/plugins/test/mocha/cli/deploy.spec.ts

```diff
@@ -718,51 +721,14 @@ const expectValidDeployment = async (
   expect(await provider.getCode(expectedContractAddress)).to.not.equal('0x')
   expect(await contract.number()).to.equal(BigInt(2))
 
-  expect(preview).to.deep.equal({
```

Comment:
It was pointless to check the preview here because it's not used for anything when it's returned by the `deploy` function. It was a maintenance burden, so I removed it from these tests. It'd make more sense to check the `actionInput` array, since that's a direct input to the preview, but it wasn't necessary for this PR, so I didn't bother

Reply:
What do you mean it's not used for anything in the deploy function? Isn't the preview displayed in the deploy command? I thought the purpose of returning and checking this here was to confirm that the preview is correct.

Reply:
Yeah thanks for fact checking me. I updated this test suite to check for the presence of the correct action inputs instead of checking for the preview since this is more comprehensive (the action inputs contain several more fields than the preview), maintainable (the action inputs are less likely to change than the structure of the preview since changing the action inputs is a breaking change), and logically equivalent (since the preview is created directly from the action inputs and the other fields that we already check in the network config)